permits/admin.py

```diff
@@ -1335,45 +1329,6 @@ def has_background_image(self, obj):
     has_background_image.short_description = "Image de fond"
 
 
-# AuthToken admin
-
-
-class TokenAdmin(BaseTokenAdmin):
```

Comment:
Unfortunately, the built-in admin for knox is not very usable (because knox doesn't store the key, but only the hash). The suggested way to generate tokens is now an API view :  http://localhost:9095/api-tokens/login/

If an admin panel is needed, we will need to add it.